troposphere/static/js/components/projects/resources/instance/details/actions/InstanceActionsAndLinks.react.js

```diff
@@ -96,7 +96,7 @@ export default React.createClass({
     render: function() {
       var webShellUrl = this.props.instance.shell_url(),
           remoteDesktopUrl = this.props.instance.vnc_url(),
-          usesRemoteDesktop = false,
+          usesRemoteDesktop = !!(this.props.instance && this.props.instance.get('vnc')),
```

Comment:
We want the `usesRemoteDesktop = !!(this.props.instance && this.props.instance.get('vnc')),` line here. 

The ultimate fix is for this to be something that can be turned on or off based on a settings - similar to the `troposphere` database / api model "site_metadata".